spec/embedded_sass.proto

```diff
@@ -1060,3 +1067,13 @@ enum CalculationOperator {
   // The division operator.
   DIVIDE = 3;
 }
+
+// The built-in Node.js Package Importer, which is a Package Importer that
+// resolves using the standards and conventions of the Node.js ecosystem. It
+// enables a `pkg:` URL scheme for usage with `@use` that directs an
+// implementation to resolve a URL within a dependency. It must be instantiated
+// with an`entry_point_url`, which is passed as the `parentURL` to the Node.js
+// resolution algorithm.
+message NodePackageImporter {
+  string entry_point_url = 1;
```

Comment:
Is there any case where this isn't going to be a `file:` URL? If not, should we make it a path instead of a URL?

Reply:
This can be a path. I've updated that in daf424181f903bfadeb37fcce494ea82623d298d